package.json

```diff
@@ -76,7 +76,7 @@
     "webpack-merge": "^3.0.0"
   },
   "scripts": {
-    "prepublish": "npm run clean && npm run build && npm run test",
+    "prepare": "npm run clean && npm run build && npm run test",
```

Comment:
We added `prepublish` task to run everything that is required for publishing, according to your link it won't change.